tachyon/c/math/elliptic_curves/point_traits_forward.h

```diff
@@ -0,0 +1,11 @@
+#ifndef TACHYON_C_MATH_ELLIPTIC_CURVES_POINT_TRAITS_FORWARD_H_
+#define TACHYON_C_MATH_ELLIPTIC_CURVES_POINT_TRAITS_FORWARD_H_
+
+namespace tachyon::c::math {
+
+template <typename Point>
+struct PointTraits;
```

Comment:
I think you didn't remove `point_traits_forward.h` in cc

Reply:
It seems to be wrong by git?
`point_traits_forward.h` was removed in this commit.